### ISSUE TITLE
No API Support anymore

### ISSUE THREAD

**Issue by Webserve** [node-red/node-red-web-nodes#259]
https://darksky.net/dev

https://blog.darksky.net/

The api ends at 2021

**Comment by knolleary** [comment]
Yup - we are aware of this. The closing of the API will mark the EOL of the darksky nodes. Nothing else we can do about that.

**Comment by knolleary** [comment]
sorry, didn't mean to close... lets use this issue to update the readme of the nodes to users are aware of the EOL
